web/src/components/pair/PairCodeFileInfo.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="file">
+    <div>
+      <div class="file-info-title">
+        <v-icon color="primary">mdi-file-outline</v-icon>
+        {{ props.file.shortPath }}
+      </div>
+

```

Comment:
I also think adding the label here (if any labels are present) would be useful here.

Reply:
I will add this

Reply:
Too prevent too much wrapping I put the extra information on a separate line:
![image](https://user-images.githubusercontent.com/6013068/186421422-7f0643eb-a906-4614-a831-13338897fbc2.png)